### PR TITLE
Updates for new TNS API requirements

### PR DESCRIPTION
**Pull request by Fingel**
The TNS api now requires that you include a User-Agent header with
values for both your "bot name" and "bot id", and will return a 403 if
they are not provided. Why they can't just associate this info with your
API key?? Who knows.

### CODE REVIEW

File: tom_alerts/brokers/tns.py [hunk 1]
```diff
@@ -82,6 +82,15 @@ class TNSBroker(GenericBroker):
     name = 'TNS'
     form = TNSForm
 
+    @classmethod
+    def tns_headers(cls):
```
Comment (dmcollom) [inline tom_alerts/brokers/tns.py:86]: Can you just comment a link to the documentation where this is listed? I'm assuming it's the ``API - Search/Get objects`` section [here](https://www.wis-tns.org/content/tns-getting-started). And just to double-check, the documented code has a single pair of braces as opposed to the double pair--does it work either way?

**Review by dmcollom — APPROVED** [review]
Sorry for the delay--was on vacation last week. Just the one request for the documentation, and Lindy and I unfortunately muddied the PR a little bit by changing the target branch. Thanks for the change!

**Review by phycodurus — APPROVED** [review]

File: tom_alerts/brokers/tns.py (hunk 1 above)
Comment (Fingel) [reply tom_alerts/brokers/tns.py:86]: I actually found it in some obscure news article they posted:
https://www.wis-tns.org/content/tns-newsfeed#comment-wrapper-23710
It's not in the actual API docs from what I can tell: https://www.wis-tns.org/sites/default/files/api/TNS_APIs_manual.pdf

Comment (Fingel) [reply tom_alerts/brokers/tns.py:86]: For the record - the double braces is to escape the braces, the final string only has one.

**Pull request merged**